<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mail</class>
 <widget class="QDialog" name="mail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>264</y>
     <width>452</width>
     <height>143</height>
    </rect>
   </property>
   <property name="title">
    <string>Recipients</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QTextEdit" name="teRecipients_2">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;first@mail_server&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;second@mail_server&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>413</y>
     <width>452</width>
     <height>58</height>
    </rect>
   </property>
   <property name="title">
    <string>Subject</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_15">
    <item>
     <widget class="QLineEdit" name="leSubject_2">
      <property name="text">
       <string>Mail subject</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>8</y>
     <width>452</width>
     <height>58</height>
    </rect>
   </property>
   <property name="title">
    <string>SMTP Server</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <widget class="QLineEdit" name="leSmtpServer_2">
      <property name="text">
       <string>smtp.gmail.com</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>477</y>
     <width>452</width>
     <height>144</height>
    </rect>
   </property>
   <property name="title">
    <string>Message</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_17">
    <item>
     <widget class="QTextEdit" name="teMessage_2">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;Message from&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;Qt Simple SMTM&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;E-mail sender&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>72</y>
     <width>452</width>
     <height>58</height>
    </rect>
   </property>
   <property name="title">
    <string>Sender</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_13">
    <item>
     <widget class="QLineEdit" name="leSender_2">
      <property name="text">
       <string>user@mail_server</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>627</y>
     <width>452</width>
     <height>143</height>
    </rect>
   </property>
   <property name="title">
    <string>Files</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTextEdit" name="teFiles_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_14">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pbAddFiles_2">
        <property name="text">
         <string>Add files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbRemoveFiles_2">
        <property name="text">
         <string>Remove files</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pbSendEmail">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>782</y>
     <width>452</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLabel" name="lStatus">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>816</y>
     <width>452</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>136</y>
     <width>452</width>
     <height>58</height>
    </rect>
   </property>
   <property name="title">
    <string>Password</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_16">
    <item>
     <widget class="QLineEdit" name="lePassword_2">
      <property name="text">
       <string>your password</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>452</width>
     <height>58</height>
    </rect>
   </property>
   <property name="title">
    <string>Sender name</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <widget class="QLineEdit" name="leSenderName_2">
      <property name="text">
       <string>John</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
